ibis/tests/expr/test_value_exprs.py

```diff
@@ -350,6 +350,20 @@ def test_notnull(table):
     assert isinstance(expr.op(), ops.NotNull)
 
 
+@pytest.mark.parametrize(
+    "value",
+    [
+        param(lambda: None, id="none"),
+        param(lambda: ibis.NA, id="NA"),
+        param(lambda: ibis.literal(None, type="int32"), id="typed-null"),
+    ],
+)
+def test_null_eq_and_ne(table, value):
+    other = value()
+    assert (table.a == other).equals(table.a.isnull())
+    assert (table.a != other).equals(table.a.notnull())
```

Comment:
```suggestion
        param(None, id="none"),
        param(ibis.NA, id="NA"),
        param(ibis.literal(None, type="int32"), id="typed-null"),
    ],
)
def test_null_eq_and_ne(table, value):
    assert (table.a == value).equals(table.a.isnull())
    assert (table.a != value).equals(table.a.notnull())
```

Doesn't seem like the `lambda`s are necessary here.